stores/LayoutMumble.context.ts

```diff
@@ -3,6 +3,11 @@ import { ELayoutKind } from '@/providers/LayoutMumble.provider';
 
 export interface ILayoutContextOptions {
   setLayoutKind: React.Dispatch<React.SetStateAction<ELayoutKind>>;
+
+  // these 2 properties would need to be part of a general helpers provider
```

Comment:
perhaps add a todo here?

Reply:
well, maybe it will be like a todo that will never implemented. 
I would keep it like this atm. maybe we could rename the provider. that one would be better.